run_project_tests.py

```diff
@@ -356,6 +356,13 @@ def have_d_compiler():
         return True
     return False
 
+def have_objc_compiler():
+    if shutil.which("objc") and shutil.which("objcpp"):
+        return True
```

Comment:
This does not work. On debianlike machines there is no executable called `objc`. You just need to know that `gcc` masquerades as objc.